atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/StreamStoreRenderer.java

```diff
@@ -283,12 +285,11 @@ private void storeBlock() {
                     }
                     line("} catch (RuntimeException e) {");
                     {
-                        line("log.error(");
-                        line("        \"Error storing block {} for stream id {}\",");
+                        line("throw new SafeRuntimeException(");
+                        line("        \"Error storing block for stream\",");
+                        line("        e,");
                         line("        SafeArg.of(\"blockId\", row.getBlockId()),");
-                        line("        SafeArg.of(\"id\", row.getId()),");
-                        line("        e);");
-                        line("throw e;");
+                        line("        SafeArg.of(\"id\", row.getId()));");
```

Comment:
nit: eh, maybe obvious, but I'd prefer `streamId` for these args once the context from the `{}` placeholders is gone